src/Security/Member.php

```diff
@@ -1705,6 +1705,12 @@ public function getHtmlEditorConfigForCMS()
         $currentName = '';
         $currentPriority = 0;
 
+        // If we don't have a custom config, no need to look in all groups
+        $editorConfigMap = HTMLEditorConfig::get_available_configs_map();
+        if (count($editorConfigMap) === 1 && isset($editorConfigMap['cms'])) {
+            return 'cms';
+        }
```

Comment:
I don't think the `cms` named config is necessarily relevant. I think it's cleaner to say `if (count($editorConfigMap) === 1) { return <the only config>; }` - it doesn't matter what it's named, all that matters is that it's the only config.

Reply:
good idea! i'll update the code right away